numbers.js

```diff
@@ -1,10 +1,10 @@
 //This file contains Javascript functions that is operated on numbers.
 
 function fibo(range){
-    var f1=0
-    var f2=1
-    var f3;
-    var arr=[0,1]
+    let f1=0
+    let f2=1
+    let f3;
+    let arr=[0,1]
     for(var i=2;i<range;i++){
```

Comment:
Nice try, but always have a habit of carefully checking the code once the reviewer requests any changes, figure out the changes that has to be done

Reply:
fixed it! sorry for my mistake